staging/src/k8s.io/apiserver/pkg/audit/union.go

```diff
@@ -55,3 +58,11 @@ func (u union) Shutdown() {
 		backend.Shutdown()
 	}
 }
+
+func (u union) String() string {
+	var backendStrings []string
+	for _, backend := range u.backends {
+		backendStrings = append(backendStrings, fmt.Sprintf("%s", backend))
```

Comment:
nit: why not backend.String() instead of `printf("%s", ...)` ?

Reply:
The backend interface doesn't have string. I could add String() (or just Name()) to the interface, or try to cast to `fmt.Stringer` here, but I thought the `Sprintf` was simpler. I don't have any real preference though...

Reply:
it's fine as it is then.